CWE/inventions/New inventions.txt

```diff
@@ -300,4 +300,465 @@ full_body_replacement = {
+		modifier = {
+			factor = 1
+			OR = { continent = europe continent = north_america }
+		}
+		modifier = {
+			factor = -1
+			OR = { ruling_party_ideology = communist ruling_party_ideology = communist_social }
+		}
+		modifier = {
+			factor = -2
+			NOT = { year = 1960 }
+		}
+		modifier = {
+			factor = 1
+			has_country_flag = eurocommunism
+		}
+	}
+	effect = {
+		social_reform_desire = 0.01
+		rebel_org_gain = {
+			faction = communist_rebels
+			value = 0.05
+		}
+	}
+}
+
+eurocommunism_inv = { 
+	limit = { seventies_subcultures  = 1 } #Progressivism
+	chance = {
+		base = 1
+		modifier = {
+			factor = 1
+			OR = { government = democracy government = democracy1 government = hms_government government = hms_government1 government = socialist_democracy }
+		}
+		modifier = {
+			factor = -1
+			NOT = { continent = europe }
+		}
+		modifier = {
+			factor = -1
+			OR = { ruling_party_ideology = communist ruling_party_ideology = communist_social }
+		}
+		modifier = {
+			factor = 3
+			has_country_flag = eurocommunism
+		}
+	}
+	effect = {
+		social_reform_desire = 0.01
+	}
+}
+
+human_immunocontraception = { 
+	limit = { genomics = 1 } #AI Guided Manufacturing
+	chance = {
+		base = 1
+		modifier = {
+			factor = -1
+			OR = { ruling_party_ideology = traditionalist ruling_party_ideology = populist ruling_party_ideology = nationalist }
+		}
+	}
+	effect = {
+        	pop_growth = -0.0001
+	}
+}
+
+reversible_inhibition = {
+	limit = { clinical_pharmacology = 1 } #Advanced Biotechnology
+	chance = {
+		base = 1
+		modifier = {
+			factor = -1
+			OR = { ruling_party_ideology = traditionalist ruling_party_ideology = populist ruling_party_ideology = nationalist }
+		}
+	}
+	effect = {
+        	pop_growth = -0.0001
+	}
+}
+
+sustainable_population = {
+	limit = { globalization = 1 } #Political Globalisation
+	chance = {
+		base = 0
+		modifier = {
+			factor = -1
+			NOT = { year = 1995 }
+		}
+		modifier = {
+			factor = -1
+			OR = { ruling_party_ideology = traditionalist ruling_party_ideology = populist ruling_party_ideology = nationalist }
+		}
+		modifier = {
+			factor = 2
+			OR = { 
+				has_country_modifier = overpopulated_nation1
+				has_country_modifier = overpopulated_nation2
+				has_country_modifier = overpopulated_nation3
+				has_country_modifier = overpopulated_nation4
+				has_country_modifier = overpopulated_nation5
+				has_country_modifier = overpopulated_nation6
+				has_country_modifier = overpopulated_nation7
+				has_country_modifier = overpopulated_nation8
+			}
+		}
+		modifier = {
+			factor = 3
+			OR = {
+				has_country_modifier = overpopulated_nation10
+				has_country_modifier = overpopulated_nation11
+				has_country_modifier = overpopulated_nation12
+				has_country_modifier = overpopulated_nation13
+				has_country_modifier = overpopulated_nation14
+				has_country_modifier = overpopulated_nation15
+				has_country_modifier = overpopulated_nation16
+			}
+		}
+	}
+	effect = {
+        	pop_growth = -0.0001
```

Comment:
Sorry, I don't allow techs to affect population growth as that means I have to rebalance the pop growth system again.

Reply:
Fair, I had made these before all that. I can remove them.